source/dynamical_systems/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 set(LIBRARY_NAME dynamical_systems)
 
-find_library(state_representation REQUIRED)
```

Comment:
Was this the breaking line? Very curious. Do you have an idea what the error was?